cmd/root.go

```diff
@@ -73,12 +73,12 @@ A mutating webhook for Kubernetes, pointing the images to a new location.`,
 
 		imageSwapPolicy, err := types.ParseImageSwapPolicy(cfg.ImageSwapPolicy)
 		if err != nil {
-			log.Err(err)
+			log.Err(err).Send()
```

Comment:
I would prefer having log.Err(err).Msg("parse image swap policy failed") instead of a empty error without context. (same for below)

Reply:
@darkweaver87 I agree with @project0 ... This would allow to avoid using `Send()`

see https://github.com/rs/zerolog/blob/164f7aa1a64acb4ec010b7c0d94f6066780a3a91/event.go#L111